enhancements/node-artifacts/certificates-and-pull-secrets.md

```diff
@@ -0,0 +1,199 @@
+---
+title: node-artifacts-during-build-and-image-stream-import
+authors:
+  - "@rmarasch"
+reviewers:
+  - "@dmage"
+  - "@bparees"
+  - "@adambkaplan"
+approvers:
+  - "@dmage"
+  - "@bparees"
+  - "@adambkaplan"
+creation-date: 2019-12-02
+last-updated: 2019-12-03
+status: provisional
+---
+
+# Using node certificates/pull secrets during build and image stream import
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Allow OpenShift users to import and use images from any registry configured
+during or after the cluster installation by sharing node's artifacts(certificates
+and pull secrets) with `openshift-api` and `builder` pods.
+
+## Motivation
+
+To have image stream and builds working closely to node artifacts avoids
+complexity and redundancy as each component won't need to mount different
+Secrets and ConfigMaps to have access to information that is already present
+on node filesystem.
+
+In order to provide a seamless developer experience OpenShift should allow
+users to import or to use during builds any registry data provided during or
+after the installation, among the registries we would like to allow users to
+interact with is `registry.redhat.io`. 
+
+Nowadays credentials and additional certificates provided during cluster install
+are available on node's filesystem. If user attempts to import an image stream
+from these locations OpenShift fails as neither `openshift-api` nor `builder`
+use the credentials provided during the installation. When users configure
+additional certificates for their own registry these are also copied into node's
+filesystem.
+
+### Goals
+
+- Allow users to import image streams from images hosted in any registry provided
+  during or after cluster installation, including `registry.redhat.io`.
+- Allow users to run `builds` based on images hosted in any registry provided
+  during or after cluster installation without providing any extra credentials.
+
+### Non-Goals
+
+- Change the way users manage cluster certificates/pull secrets.
+
+## Proposal
+
+### User Stories
+
+As an OpenShift user
+I want to be able to import image streams from any image registry provided
+during or after the cluster installation
+So that I can use them without manually creating any extra credentials
+
+As an OpenShift user
+I want to be able to use, during builds, images from any registry provided
+during or after the cluster installation
```

Comment:
I would rephrase this as "i want to be able to use, during builds, images from any image registry that the cluster nodes can pull images from"

Reply:
Good improvement, tks.